lib/msal-react-native-android-poc/MSALModule/android/src/main/java/com/reactlibrary/MSALModule.java

```diff
@@ -89,6 +97,47 @@ public void onCancel() {
         };
     }
 
+     /**
+     * getAccount(): retrieves account currently signed in
+     * Parameters: Promise promise (resolve will return the account as a map if it exists; reject will return null for nonexistent map or error)
+     */
+    @ReactMethod
+    public void getAccount(Promise promise) {
+        //first load account and get status string
+        String message = loadAccount();
+        if (mAccount == null) {
```

Comment:
I would try to avoid using a global mAccount and rather move your logic from `loadAccount` into this function so you can return it directly.

Reply:
I rewrote loadAccount() to return the account directly. Logging will be handled by loadAccount(), and getAccount() will reject with a message to check the Android console if the returned account is null.

Reply:
Changes look great!